packages/teams-js/src/public/sharing.ts

```diff
@@ -52,13 +52,15 @@ export namespace sharing {
    * Opens a share dialog for web content
    *
    * @param shareWebContentRequest - web content info
-   * @param callback - optional callback
+   * @returns Promise that will be fulfilled when the operation has completed
    *
    * @internal
    */
   export function shareWebContent(shareWebContentRequest: IShareRequest<IShareRequestContentType>): Promise<void>;
   /**
-   * @deprecated with TeamsJS v2 upgrades
+   * @deprecated
+   * As of 2.0.0-beta.1, please use {@link sharing.shareWebContent sharing.shareWebContent(shareWebContentRequest: IShareRequest\<IShareRequestContentType\>): Promise\<void\>} instead.
```

Comment:
This should actually be "As of 2.0.0-beta.3, ..." since the next release will be 2.0.0-beta.3 that contains this.

Reply:
Done